Makefile

```diff
@@ -23,8 +23,11 @@ install: all
 	  install -m 0644 -D -t $(DESTDIR)/usr/lib/dracut/modules.d/$${bn} $$x/*; \
 	done
 	chmod a+x $(DESTDIR)/usr/lib/dracut/modules.d/*/*.sh $(DESTDIR)/usr/lib/dracut/modules.d/*/*-generator
+	install -m 0644 -D -t $(DESTDIR)/usr/lib/systemd/system systemd/ignition-delete-config.service
 	install -m 0755 -D -t $(DESTDIR)/usr/lib/dracut/modules.d/30ignition bin/$(GOARCH)/ignition
 	install -m 0755 -D -t $(DESTDIR)/usr/bin bin/$(GOARCH)/ignition-validate
+	install -m 0755 -d $(DESTDIR)/usr/sbin
+	ln -sf ../lib/dracut/modules.d/30ignition/ignition $(DESTDIR)/usr/sbin/ignition-rmcfg
```

Comment:
If we don't expect users to directly interact with this tool, let's put this in `/usr/libexec` instead?